_posts/2017-03-17-getting-rest-right.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Getting REST right
+author: jake_hall
+---
+
+It seems everyone is building a REST API these days - this is great! It's not easy, but there are a few key things you can do to make it easy for your consumers of your API.
```

Comment:
Is it worth talking a little more about why I'd want to use REST over say SOAP RPC? Should I just automatically use REST for everything now?